neqo-transport/src/crypto.rs

```diff
@@ -25,6 +24,7 @@ use neqo_crypto::{
     TLS_GRP_EC_SECP256R1, TLS_GRP_EC_SECP384R1, TLS_GRP_EC_SECP521R1, TLS_GRP_EC_X25519,
     TLS_GRP_KEM_MLKEM768X25519, TLS_VERSION_1_3,
 };
+use rustc_hash::FxHashMap as HashMap;
```

Comment:
Do we really see a performance gain in unit test runtime? Is it worth it?

Reply:
I think we do. I'm currently mucking with the bencher setup so we get deltas-against-main for the performance table at the bottom of the benchmarks,

Reply:
So the changes in THIS file relate to CryptoStates::initials, which might be better served by using EnumMap.